travis-ci/cleanup-integration-tests.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+
```

Comment:
You'll probably want `set -e` here so that, if a command fails, the entire script fails.

Reply:
👍